20-intel.conf

```diff
@@ -0,0 +1,6 @@
+Section "Device"
+   Identifier  "Intel Graphics"
+   Driver      "modesetting"
+   Option      "monitor-eDP-1" "eDP-1"
```

Comment:
Did you test connecting/disconnecting external monitors with this fix? I may be wrong, but i have a feeling it may negatively affect monitor hotplugging.

Reply:
For you, I tried with the old monitor and it works just as intended. I just cannot manage to change scaling per monitor but I don't know whether it's the Xorg's problem or my fix.

Reply:
OK then. I was worried the fix could break some existing behaviour, in which case we needed to warn users about it. If it doesn't, all is good.